llvm/lib/Target/AIE/AIEMachineScheduler.cpp

```diff
@@ -639,7 +640,7 @@ void AIEPostRASchedStrategy::commitBlockSchedule(MachineBasicBlock *BB) {
 
   // Safety margin, swp epilogue
   // Note that the prologue is handled in a different way. See enterMBB.
```

Comment:
Comment is out of date, we now only handle the safety margin here.

Reply:
Curious: Can we have both a safety margin and a top-fixed region? If not, can we assert it doesn't happen?

Reply:
We cannot have! If we need to supply safety margin for swp loop, it means an incorrect schedule. I will insert an assert.